src/backend/joanie/core/permissions.py

```diff
@@ -19,3 +21,17 @@ def has_object_permission(self, request, view, obj):
         """Check permission for a given object."""
         abilities = obj.get_abilities(request.user)
         return abilities.get(request.method.lower(), False)
+
+
+class HasAPIKey(permissions.BasePermission):
+    """Permission class to grant access to our remote endpoints API."""
+
+    def has_permission(self, request, view):
+        """
+        Check if the token is set in the request and known in our settings `AUTHORIZED_API_TOKENS`.
+        """
+        token = request.headers.get("Authorization")
+        if not token:
+            return False
```

Comment:
This check is useless has `None in settings.AUTHORIZED_API_TOKENS` will return `False`.

```suggestion
```